handlers.go

```diff
@@ -311,6 +311,13 @@ func (r *resolver) handleRelations(addedApplications map[string]string) {
 
 // handleOffers populates the change set with "CreateOffer" records.
 func (r *resolver) handleOffers(addedApplications map[string]string) {
+	for name, saasSpec := range r.bundle.Saas {
+		r.changes.add(newConsumeOfferChange(ConsumeOfferParams{
+			URL:             saasSpec.URL,
+			ApplicationName: name,
+		}))
```

Comment:
@achilleasa any idea what's required here?

Reply:
Perhaps the ApplicationName field is redundant as `juju consume` works fine when provided with just the offer URL. I think it's fine to keep it as we do need it for the Description()/GUIArgs() implementations.